graphql_client_cli/README.md

```diff
@@ -1,3 +1,61 @@
 # GraphQL client CLI
 
 This is still a WIP, the main use for it now is to download the `schema.json` from a GraphQL endpoint, which you can also do with [apollo-codegen](https://github.com/apollographql/apollo-cli).
+
+## Install
+
+```
+cargo install graphql-client-cli
```

Comment:
What do you think of adding the `--force` flag? (in general users always want the latest version imo)